src/main/java/org/spongepowered/api/text/channel/MessageChannel.java

```diff
@@ -0,0 +1,181 @@
+    /**
+     * A channel with all online players, as well as the server console, as members.
+     */
+    MessageChannel TO_ALL = () -> ImmutableSet.<MessageReceiver>builder()
+            .addAll(Sponge.getGame().getServer().getOnlinePlayers())
+            .add(Sponge.getGame().getServer().getConsole())
+            .build();
+
+    /**
+     * Gets a message channel that targets all subjects with the given permission.
+     *
+     * @param permission The permission to target
+     * @return The channel
+     * @see PermissionMessageChannel
+     */
+    static MessageChannel permission(String permission) {
+        return new PermissionMessageChannel(permission);
+    }
+
+    /**
+     * Gets a message channel that targets all subjects contained within the given channels
+     * and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see CombinedMessageChannel
+     */
+    static MessageChannel combined(MessageChannel... channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets all subjects contained within the given channels
+     * and applies the message transformations of each channel in order.
+     *
+     * @param channels The channels to combine
+     * @return The channel
+     * @see CombinedMessageChannel
+     */
+    static MessageChannel combined(Collection<MessageChannel> channels) {
+        return new CombinedMessageChannel(channels);
+    }
+
+    /**
+     * Gets a message channel that targets the given sources.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(MessageReceiver... recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Gets a message channel that targets the given recipients.
+     *
+     * @param recipients The recipients
+     * @return The channel
+     * @see FixedMessageChannel
+     */
+    static MessageChannel fixed(Collection<? extends MessageReceiver> recipients) {
+        return new FixedMessageChannel(recipients);
+    }
+
+    /**
+     * Broadcast a message to this channel, transforming and sending it to
+     * the members.
+     *
+     * @param original The original text to send
+     */
+    default void broadcast(Text original) {
+        checkNotNull(original, "original text");
+        for (MessageReceiver member : this.getMembers()) {
+            this.transformMessage(null, member, original).ifPresent(member::sendMessage);
+        }
+    }
+
+    /**
+     * Send a message to this channel, transforming and sending it to the members.
+     *
+     * @param sender The sender of the message
+     * @param original The original message to send
+     */
+    default void send(Object sender, Text original) {
+        checkNotNull(sender, "sender");
```

Comment:
I'm still thinking about this one, but my thoughts currently are: if no sender is available the `broadcast` method should be sent.